safe/gui/tools/wizard/test/test_keyword_wizard.py

```diff
@@ -1031,14 +1024,14 @@ def test_integrated_line(self):
         # check if in step subcategory
         self.check_current_step(dialog.step_kw_subcategory)
 
-        expected_subcategories = ['Road']
+        expected_subcategories = ['Roads']
```

Comment:
Where does it come from ? Is it when we change in the definitions.py ? Can't you make the test dynamic ? To fetch the expected value from the definitons ?
Because we are not testing the definitions here, only the wizard, so our tests shoudn't be failing because of a change in definitions.